windows.go

```diff
@@ -21,7 +21,7 @@ const (
 	WUStatusAborted
 )
 
-type WindowsUpdateWatcher struct {
+type windowsUpdateWatcher struct {
```

Comment:
Why change to unexported?

Reply:
Yeah I get the idea. But @golangcibot is kinda right here, that it's annoying to use an unexported type. Code completion might not work and if the type is not exported it should only be used internally. So I would keep it exported.

If in some scenario you want to initialise the struct by yourself you're free to do so. Using the function is fine to be the intended way to get an instance of the struct. Maybe add a comment.

Reply:
I rewrite this thing when I move all the watchers into separate pkgs. I just needed to refactor this file first so this is a very temp solution.

Can we merge it and I will propose moving PR that will have exported structs?

Reply:
> Can we merge it and I will propose moving PR that will have exported structs?

yeah we can do this, I just wonder why we change it from exported to unexported and then change it back in another PR. Couldn't we just skip the hassle, leave it exported and still move into its own package in another PR? Or do I miss something?

Reply:
because in case of fsWatcher and netWatcher they were already unexported. So I made it consistent. 
Then I realised that it will be changed to exported after we will move to pkgs

Reply:
Aahhhh gotcha.
I would be fine either way but if we have to change something anyway what about export them all and be done with that?

Reply:
Done 😁